### ISSUE TITLE
[3.6.3, 3.5.2] Dragonbones error & "animationPose" is undefined on Android platform

### ISSUE THREAD

**Issue by mrtzd** [cocos/cocos-engine#13467]
### Cocos Creator version

3.6.3, 3.5.2

### System information

Windows 10 Editor

### Issue description

I have a 2D game where the character has an idle animation and the user controls the character which can shoot arrows using a bow. This works perfectly when the game is running in the browser without any errors:

https://user-images.githubusercontent.com/82378750/203154045-fc9b3688-5d42-4adf-8352-0f39bb84214c.mp4

But running the game on an Android device and then trying to shoot an arrow I get errors in the log and the animation is broken. Here is part of the code that causes this error (line 154 of 'Player.ts' script in demo):

```
this.aimArm.offset.x = this.aimArmX - (this.aimArm.origin.x + this.aimArm.animationPose.x);
this.aimArm.offset.y = this.aimArmY + this.aimArm.origin.y + this.aimArm.animationPose.y;

this.aimArm.invalidUpdate();
```

Also I've noticed that on Android the value of "animationPose" of any bone is always undefined and so I can't use it but even without using the "animationPose" the same error happens.

### Relevant error log output

This was the log output running on an Android device (v3.6.2):

```
D/Cocos: 22:45:05 [DEBUG]: D/ JS: [ERROR]: TypeError: Cannot read property 'x' of undefined
    TypeError: Cannot read property 'x' of undefined
        at Player.animateArms (assets/main/index.js:636:99)
        at Player.updateCharacter (assets/main/index.js:714:18)
        at Player.lateUpdate (assets/main/index.js:733:18)
        at eval (eval at createInvokeImplJit (src/cocos-js/cc.js:34062:40), <anonymous>:3:65)
        at ReusableInvoker._invoke (src/cocos-js/cc.js:34069:19)
        at ReusableInvoker.invoke (src/cocos-js/cc.js:34051:22)
        at ComponentSc
```

This is the same error log for the v3.5.2 of this demo:

```
D/jswrapper: JS: [ERROR]: TypeError: Cannot read property 'x' of undefined
    TypeError: Cannot read property 'x' of undefined
        at Player.animateArms (assets/main/index.js:636:99)
        at Player.updateCharacter (assets/main/index.js:714:18)
        at Player.lateUpdate (assets/main/index.js:733:18)
        at eval (eval at createInvokeImplJit (src/cocos-js/cc.js:47540:40), <anonymous>:3:65)
        at ReusableInvoker._invoke (src/cocos-js/cc.js:47547:19)
        at ReusableInvoker.invoke (src/cocos-js/cc.js:47529:22)
        at ComponentScheduler.lateUpdatePhase (src/cocos-js/cc.js:47683:40)
        at Director.tick (src/cocos-js/cc.js:51632:41)
        at callback (src/cocos-js/cc.js:51058:30)
        at tick (jsb-adapter/jsb-builtin.js:601:7)
```

### Steps to reproduce

Simply run the attached demo project on an Android device and then swipe on the screen to see the error (by swiping you pull the string of the bow).

**Comment by zhakesi** [comment]
native Transform animationPose not export to js

**Comment by mrtzd** [comment]
> native Transform animationPose not export to js

Sorry to bother you, so will this issue get fixed in the next release? I was getting ready to finish my game and release it in the near future but this is a major blocking issue and I'm not aware of any workarounds.

**Comment by mrtzd** [comment]
@zhakesi  I've tested this issue again and seems like it is fixed in v3.7.0 since "animationPose" is no longer undefined on Android. I don't know which pull request was responsible for this fix so I'll leave it open. Anyway thanks for fixing this!

**Comment by mrtzd** [comment]
Fixed by #12410
